terra_layer/urls/__init__.py

```diff
@@ -17,6 +18,7 @@
     # Extra urls from third part modules
     path("", include("terra_accounts.urls")),
     path("", include("terra_utils.urls")),
+    path("", include("mapbox_baselayer.urls")),
```

Comment:
What do you think of letting the module baselayer include his urls by himself ?

Reply:
what do you think @submarcos ?

Reply:
It depends on project policy. Its is just a non mandatory helper to provide an unauthenticated way to serve Base layer tilejson. If project require more protection, it is easy to serve these urls in an authenticated view. But I think mapbox and map js frameworks dont like JWT

Reply:
For the moment, we can keep it here. We'll see if we need to change.